fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,7 +1,22 @@
-Are you tired of searching for your plastic reward card while checking out at the store? Are you looking for a free solution which will not take your information?
+Stop the search for plastic reward cards during store or webshop checkout.
+Scan barcodes for deals to your device using its camera, and keep not remembering where they are.
```

Comment:
"and keep not remembering where they are"?

Reply:
The cards. That didn't work.